respeaker_ros/cfg/Respeaker.cfg

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+#
+# WARNING!!
+# This file is automatically generated.
+# DO NOT MODIFY THIS FILE!
+#
+# To generate this file, please run:
+# cd /path/to/this/package
+# python ./scripts/respeaker_gencfg.py
```

Comment:
please add when we need to run `./scripts/respeaker_gencfg.py` in README.md or, may be you 'd better to use cmake to generate this file.

Reply:
@k-okada This script is to get default value from the device as current values.
That is what I thought once before, but it requires device connected on build time.
I added an instruction to the readme.

Reply:
I see, so add how to use this script and when we need to generate this file

Reply:
to README.md